### ISSUE TITLE
Chapter 10: Question - Instance method and static method with the same name

### ISSUE THREAD

**Issue by ClimberBear** [PacktPublishing/Web-Development-with-Blazor-Third-Edition#18]
This is not an issue, only a question.

It's the first time that I see an instance method with the same name than statice method. Even, I believed that this was not possible.

Does it make any sense or need to have the both method with the same name (ShowAlert) in the JSInterop / WebAssembly sample.

Thanks in advance.

**Comment by EngstromJimmy** [comment]
Are you referring to the method in the Razor file and in the CS?
They don't have to be named the same thing since one of them has an argument; it is not the same method signature.

**Comment by ClimberBear** [comment]
Yes, the both methods have different signature, but the same name. This is confusing, more when they are declared in different files, have differenet purposes (JavaScript interface vs. event method) but belong to the same class. It's only a personal opinion, but I have always found the naming conventions and forced naming rules in C# / .Net Core confusing.
